#167749943 Setup unit testing

ibraheem-adeyemo opened the pull request:
#### What does this PR do?
- Implement unit testing

#### Description of task to be completed
- Have npm test script working
- Have app.test.js created
- Test http://localhost:3000/

#### How should this be manually tested
- After cloning the repo cd into it and run npm install and npm test

#### What are the relevant pivotal tracker stories


[#167749943](https://www.pivotaltracker.com/story/show/167749943)

davidteejay reviewed (changes requested):
For the test and the dev scripts, we should set the node environment in the command. For the test script you should add `NODE_ENV=test` and `NODE_ENV=development` for dev

ibraheem-adeyemo committed:
Chore(testing): Add unit testing directory to the codebase

 - test localhost:3000/
 - Add test script to  package.json
 - Add start script to the package.json
 - Add build script to the package.json

[Maintain #167749943n #167995313]
